velox/expression/tests/ExprEncodingsTest.cpp

```diff
@@ -360,46 +360,75 @@ class ExprEncodingsTest
   void run(
```

Comment:
nit: update comment for `run()` to add that it also performs an iteration which clears all caches utilized during expression evaluation, such as shared sub-expressions, memoization, vector pool, and error vector, and then confirms that the memory usage before and after evaluation is consistent.